### ISSUE TITLE
No artifact images leading to no images at all?

### ISSUE THREAD

**Issue by KenWigs** [FarbstoffRSL/RSL-Helper#145]
**Bug describtion / Fehlerbeschreibung**

No artifact images are found in either cPictures or Skills. A mysterious temporary file ("um28459494_Artefacts.tmp") exists in the main folder. RSL Helper was installed using admin rights. RSL Helper was updated to V2.7, Release 20, Build 7 using admin rights. I have never gotten images (champion or skills)  to appear in my RSL Helper main window since I have been using the program since V2.6 - Release 8 - Build 21. Randomly and rarely, when farming, an artifact image will appear in the "Last Rewards" portion of the main window.

cPictures is full of champion portraits (686 items). Skills contains 1899 items.

**HowTo Reproduce / Wie man es reproduziert**

Just starting the program. Even after running any content, images do not appear.

**Expected behavior / Erwartetes verhalten**

**System(please complete the following information):**
 
Images to appear in the main window. Champion index to show champion pictures and can search by name or use the available filters.

**Game Info(please complete the following information):**

 V2.7, Release 20, Build 7

**Additional context / Weiterer Infos**

I wonder if error in displaying artifacts is causing errors in other parts of the program. Only autoclicker seems to work well. Champion Index can filter by level, but all I see are champion ID numbers. Searching by faction, name, affinity, or rarity shows no results. Also clan boss simulator, I cannot select any champions.
Event tracker mostly works, but if an error occurs in tracking an event, the tracker can not track that specific event. The tracker can track other events just fine.

StartLog starts here:
Welcome to the World of Colour :)
RSL Helper V2.7 - Release: 20 - Build: 7
developed by Farbstoff RSL - Germany
**********************************************
Check for Admin Rights - Done
**********************************************
Check if a main Update available
CurrentVersion: 27207
OnlineVersion: 27207
Check if a new Updater available - Done
**********************************************
Load game staticdata - Done
**********************************************
Set values - Done
**********************************************
LoadFileDir
Champion Pictures Dir found: C:\Program Files\RSL_Helper_X64\cPictures\
***********************P***********************
FindAllFiles - Done
686 Files found
**********************************************
Load Champion Pictures - Done
**********************************************
Load Skill Pictures - Done
**********************************************
Check for Directory and File Rights - Done
**********************************************
Add game Images - Done
Search Process: Raid.exe - Not Found
**********************************************
Start Raid.exe from settings path
Wait for the Window from RSL - Found
Start Worker Threads - Done
Eject Old DLL Imports - Done
Inject DLL in Raid.exe - Done
**********************************************
Create Settings - Done
Create Autoklicker - Done
Create ChampionIndex - Done
Create Tools - Done
Create CBSimulator - Done
Create ChampionDetailView - Done
Create ShardTracker - Done
Create EventTracker - Done
Create AutoSellSetups - Done
**********************************************
Open RSL Helper - Have Fun :)

**Comment by Dwza** [comment]
have you tried to fully uninstall the helper, restart pc, install latest version with admin rights and check if it still happens?

**Comment by KenWigs** [comment]
Yes, sometime early in version 2.7. I used the uninstaller (with admin rights). Then deleted the entire directory. Force quit all instances of Plarium Play through task manager. Then installed using admin rights. I will try again now, adding the restart.

**Comment by KenWigs** [comment]
Okay, used uninstaller (w admin), deleted directory, cleaned registry, rebooted, downloaded latest RSL Helper Installer, ran it with admin. Ran updater with admin. Ran application with admin. Did a quick campaign run. No images.

StartLog below:
Welcome to the World of Colour :)
RSL Helper V2.7 - Release: 20 - Build: 7
developed by Farbstoff RSL - Germany
**********************************************
Check for Admin Rights - Done
**********************************************
Check if a main Update available
CurrentVersion: 27207
OnlineVersion: 27207
Check if a new Updater available - Done
**********************************************
Load game staticdata - Done
**********************************************
Set values - Done
**********************************************
LoadFileDir
Champion Pictures Dir found: C:\Program Files\RSL_Helper_X64\cPictures\
***********************P***********************
FindAllFiles - Done
686 Files found
**********************************************
Load Champion Pictures - Done
**********************************************
Load Skill Pictures - Done
**********************************************
Check for Directory and File Rights - Done
**********************************************
Add game Images - Done
Search Process: Raid.exe - Not Found
**********************************************
Start Raid.exe from settings path
Wait for the Window from RSL - Found
Start Worker Threads - Done
Eject Old DLL Imports - Done
Inject DLL in Raid.exe - Done
**********************************************
Create Settings - Done
Create Autoklicker - Done
Create ChampionIndex - Done
Create Tools - Done
Create CBSimulator - Done
Create ChampionDetailView - Done
Create ShardTracker - Done
Create EventTracker - Done
Create AutoSellSetups - Done
**********************************************
Open RSL Helper - Have Fun :)

Image below:
![RSL1](https://user-images.githubusercontent.com/89710650/133531134-f8e3ab45-e26d-4c8b-815b-70cac6fa951c.jpg)

**Comment by Cyberion71** [comment]
I have a similar issue. Everything was working perfectly until just recently but now the Champion Index is unpopulated and no images or stats appear when battles are running. Autoclicker will only run the 1st battle then nothing. Have uninstalled, deleted folders / reinstalled with admin permissions, rebooted, nothing changes. Both cPictures and Skills folders are full of images. Almost like something in the last update broke it.

![ChampIndex](https://user-images.githubusercontent.com/90920669/133808214-ae710b43-e212-4f0b-b8c3-221e733f5d05.png)
![NoChampData](https://user-images.githubusercontent.com/90920669/133811793-58092aa3-7323-4592-b154-211c35a32e9c.png)
![mtclicker](https://user-images.githubusercontent.com/90920669/133811858-3a2f842e-592d-4b21-a803-0ecae3e6d4f4.png)

**Comment by KenWigs** [comment]
Used the updater (with admin) to update to newest version. Still no images after running a campaign run.
![Raid2](https://user-images.githubusercontent.com/89710650/133862020-e56e5695-b93f-4fb7-9436-409dcafb9e59.jpg)

**Comment by KenWigs** [comment]
Upgraded to  V2.7 - Release 32 - Build 2.
Still no images.

**Comment by Dwza** [comment]
What is your system language? I guess you are using windows 10 ?!

**Comment by KenWigs** [comment]
Yes, Windows 10. System is English (United States)

**Comment by Dwza** [comment]
Afte the updates... is this problem still present?

**Comment by KenWigs** [comment]
I just tried to update using the updater (with admin). The updater currently does not see the update and thinks my version is the newest.

**Comment by Dwza** [comment]
arte you in discord as well?

**Comment by KenWigs** [comment]
No, but I can join. It will have to be when I get home from work, later today.

**Comment by Dwza** [comment]
ok, perheps the help there is bigger than here, than it seems like a problem that have to be tested
